src/app/state/paratimes/types.ts

```diff
@@ -23,7 +23,10 @@ export interface TransactionForm {
   confirmTransferToValidator: boolean
   confirmTransferToForeignAccount: boolean
   defaultFeeAmount: string
+  // compatible with oasisRT.signatureSecp256k1
   ethPrivateKey: string
+  // provided by user and used in form inputs allowing back and forth form navigation
+  ethPrivateRawKey: string
```

Comment:
(I'd name it ethPrivateKeyRaw so it would show up in the searches I did a few months ago, looking for leaks of anything named `privatekey`)